docs/csharp/misc/cs0021.md

```diff
@@ -1,52 +1,52 @@
 ---
 title: "Compiler Error CS0021"
 ms.date: 09/26/2019
-f1_keywords: 
+f1_keywords:
   - "CS0021"
-helpviewer_keywords: 
+helpviewer_keywords:
   - "CS0021"
 ms.assetid: 4eb5fa24-8261-4962-b36a-224be5074217
 ---
 # Compiler Error CS0021
 
-Cannot apply indexing with [] to an expression of type 'type'  
-  
-An attempt was made to access a value through an indexer on a data type that does not support [Indexers](../programming-guide/indexers/index.md).  
-  
-You may get CS0021 when trying to use an indexer in a C++ assembly. In this case, decorate the C++ class with the `DefaultMember` attribute so the C# compiler knows which indexer is the default. The following sample generates CS0021.  
-  
-## Example  
-
-The following C++ example compiles to a .dll file. Note that the the [DefaultMember](xref:System.Reflection.DefaultMemberAttribute) attribute is commented out in order to generate the error.  
-  
-```cpp  
-// CPP0021.cpp  
-// compile with: /clr /LD  
-using namespace System::Reflection;  
-// Uncomment the following line to resolve  
-//[DefaultMember("myItem")]  
-public ref class MyClassMC  
-{  
-        public:  
-        property int myItem[int]  
-        {  
-            int get(int i){  return 5; }  
-            void set(int i, int value) {}  
-        }  
-};  
-```  
+Cannot apply indexing with [] to an expression of type 'type'
+
+An attempt was made to access a value through an indexer on a data type that does not support [Indexers](../programming-guide/indexers/index.md).
+
+You may get CS0021 when trying to use an indexer in a C++ assembly. In this case, decorate the C++ class with the `DefaultMember` attribute so the C# compiler knows which indexer is the default. The following sample generates CS0021.
+
+## Example
+
+The following C++ example compiles to a .dll file. Note that the [DefaultMember](xref:System.Reflection.DefaultMemberAttribute) attribute is commented out in order to generate the error.
```

Comment:
Probably end this line with `:` instead of `.`

Reply:
The Default Member should be put between back ticks here. so it's `DefaultMember` not DefaultMember